src/yunohost/service.py

```diff
@@ -74,6 +74,10 @@ def service_add(name, status=None, log=None, runlevel=None):
     if runlevel is not None:
         services[name]['runlevel'] = runlevel
 
+
+    if need_lock:
+        services[name]['need_lock'] = True
```

Comment:
Those 2 lines can be replace by: `services[name]['need_lock'] = need_lock` :)

Reply:
Ca ne donne pas le même résultat.
Avec mon code, une paire clé valeur est ajoutée dans le fichier yml, uniquement si l'option "need-lock" est ajoutée quand on lance la fonction "service add".
Avec ton code une paire clé valeur est ajoutée dans tous les cas, mais avec une valeur différente.

It doesn't give the same result.
With my code, a value key pair is added to the yml file, only if the "need-lock" option is added when you launch the "service add" function.
With your code a value key pair is added in all cases, but with a different value.

Reply:
Agreed with Irina :+1: (though that's a small detail - I'm happy with both cases)

Reply:
Indeed, my bad :sweat_smile: